lib/tasks/cluster.rake

```diff
@@ -82,6 +82,83 @@ namespace :cluster do
     Cluster::Console.run
   end
 
+  desc Cluster::RakeDocs.new('cluster:reset').desc
+  task reset: [:configtest, :config_sync_check, :production_failsafe] do
+    recipes = %W|mh-opsworks-recipes::stop-matterhorn
```

Comment:
could this also be `Rake::Task['matterhorn:stop'].execute`. Just curious if there's a difference.